<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>epub_creator</class>
 <widget class="QMainWindow" name="epub_creator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>epub_creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="MainFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="view_and_modSp">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListWidget" name="fileView"/>
          <widget class="QSplitter" name="modSp">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>模式</string>
            </property>
            <property name="buddy">
             <cstring>ModSelection</cstring>
            </property>
           </widget>
           <widget class="QComboBox" name="ModSelection"/>
          </widget>
         </widget>
         <widget class="QSplitter" name="ctrlUnit">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="authorSp">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="Author">
            <property name="text">
             <string>作者</string>
            </property>
            <property name="buddy">
             <cstring>editAuthor</cstring>
            </property>
           </widget>
           <widget class="QLineEdit" name="editAuthor"/>
          </widget>
          <widget class="QSplitter" name="BnameSp">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="Bname">
            <property name="text">
             <string>书名</string>
            </property>
            <property name="buddy">
             <cstring>editBname</cstring>
            </property>
           </widget>
           <widget class="QLineEdit" name="editBname"/>
          </widget>
          <widget class="QSplitter" name="sqrSp">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="UpDownSp">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QPushButton" name="Up">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
            <widget class="QPushButton" name="Down">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </widget>
           <widget class="QSplitter" name="FileSp">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QPushButton" name="Addfile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>添加文件</string>
             </property>
            </widget>
            <widget class="QPushButton" name="deleteFile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>删除文件</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QPushButton" name="Clear">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
          <widget class="QPushButton" name="CreateSep">
           <property name="text">
            <string>分别生成</string>
           </property>
          </widget>
          <widget class="QPushButton" name="CreateAsOne">
           <property name="text">
            <string>合并生成</string>
           </property>
          </widget>
          <widget class="QSplitter" name="outputSp">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="output">
            <property name="text">
             <string>Dircetory</string>
            </property>
           </widget>
           <widget class="QPushButton" name="setOutput">
            <property name="text">
             <string>输出文件夹</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
